projects/systelab-virtual-keyboard/src/lib/constants.ts

```diff
@@ -7,30 +7,77 @@
  * strictly forbidden unless prior written permission is Instrumentation Laboratory Company.
  */
 
-export enum SystelabVirtualKeyboardButton {
-    Done = '{done}',
-    Enter = '{enter}',
-    Shift = '{shift}',
-    Lock = '{lock}',
-    Backspace = '{bksp}',
-    Language = '{language}',
-    Space = '{space}',
-    Tab = '{tab}',
-}
+export namespace SystelabVirtualKeyboardConstants {
 
-export enum SystelabVirtualKeyboardLayouts {
-    default = 'default',
-    shift = 'shift',
-    numeric = 'numeric',
-}
+    export enum Layouts {
+        default = 'default',
```

Comment:
I would modify this enum to fulfill the nomenclature for the symbols, so they are written using camel case (`Default` instead of `default`). Same for the other enums in this file.